code/conf/stanford.yaml

```diff
@@ -0,0 +1,15 @@
+environment:
+  name: "Stanford-v0"
+  entry_point: "environments.stanford_client:StanfordEnvironmentClient"
+  p_blank: 0.5
+  frameskips_cases: ['Asteroids']
+  vec_norm: False
+
+  model_adaptation:
+    cnn_channels: [32, 64, 32]
+    observation_type : 32x32
+    encoder_batch_norm: True
+    policy_batch_norm: True
+
+  rl_setting_adaptation:
+   gamma: 0.99 
```

Comment:
Other configs we probably want to include: learning rate, number of particles and z_dim, num_processes, num_steps, and so on.